test/Form_test.js

```diff
@@ -1123,6 +1123,46 @@ describe("Form", () => {
     });
   });
 
+  describe("ID generation", () => {
+    const schema = {
+      type: "object",
+      properties: {
+        foo: {
+          type: "string",
+          minLength: 4
+        },
+        bar: {
+          type: "string"
+        },
+      },
+    };
+
+    const formProps = {
+      schema,
+      liveValidate: true,
+    };
+
+    it("should generate ids based on the schema properties", () => {
+      const {node} = createFormComponent(formProps);
+
+      expect(node.querySelector("input#root_foo")).not.to.be.null;
+      expect(node.querySelector("input#root_bar")).not.to.be.null;
```

Comment:
The default way of generating an `id` for each form element is `root_[field_name]`, where `field_name` corresponds to each of the keys in the properties object in the schema.